lighthouse-core/fraggle-rock/gather/runner-helpers.js

```diff
@@ -76,7 +78,7 @@ async function collectPhaseArtifacts(options) {
         url: await driver.url(),
         gatherMode,
         driver,
-        dependencies,
+        dependencies: /** @type {Dependencies} */ (dependencies),
```

Comment:
can we cast the `{}` instead on line 75? don't want to lose the type of `collectArtifactDependencies` if it changes in the future